### PR TITLE
Thor

### PR DESCRIPTION
**Pull request by abdullacontractor**
Thor reskin for warehousing library. Will use these changes to create a new version and then bump rango and pacman to it.

The changes here are involved in making those two apps look like this https://www.loom.com/share/2370fcf4af504a26b7c46c352209023e

### CODE REVIEW

File: tests/dummy/app/index.html [hunk 1]
```diff
@@ -12,8 +12,7 @@
     <link rel="stylesheet" href="{{rootURL}}assets/vendor.css">
     <link rel="stylesheet" href="{{rootURL}}assets/dummy.css">
 
-    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Raleway:100,400,600">
-    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600">
+    <link href='https://d3ds1rcygteua7.cloudfront.net/assets/fonts/avenir.css' rel='stylesheet' type='text/css'>
```
Comment (selvagsz) [inline tests/dummy/app/index.html:15]: hope this s3 object id never changes

**Comment by abdullacontractor** [comment]
Ignore failing tests, travis has always been red on this repo. Many issues.

File: tests/dummy/app/index.html (hunk 1 above)
Comment (abdullacontractor) [reply tests/dummy/app/index.html:15]: @billybonks Thoughts about this? Also that css file is still missing font weight 500.

File: app/styles/@tradegecko/warehousing/variables/base-colors.scss [hunk 2]
```diff
@@ -1,77 +1,50 @@
...
-$c-prosper:            #eedf24;
-$c-prosper-dark:       #dacc21;
-
-$c-watermelon-light:   #fa7367;
-$c-watermelon:         #f6574d;
-$c-watermelon-dark:    #e14f46;
-
-$c-elite-light:        #e3c63c;
-$c-elite:              #d0aa2b;
-$c-elite-dark:         #c09146; // used to be #d0b537;
```
Comment (selvagsz) [inline app/styles/@tradegecko/warehousing/variables/base-colors.scss:77]: are we sure these deleted scss variables aren't being used in the consuming apps ?

File: tests/dummy/app/index.html (hunk 1 above)
Comment (selvagsz) [reply tests/dummy/app/index.html:15]: ah this is a dummy app. Have we made this change on pacman/rango ?

File: app/styles/@tradegecko/warehousing/variables/font-variables.scss [hunk 3]
```diff
@@ -0,0 +1 @@
+$sans-font-family: 'AvenirNext', Helvetica, Arial, sans-serif;
```
Comment (selvagsz) [inline app/styles/@tradegecko/warehousing/variables/font-variables.scss:1]: we should better name it as `$font-family`

File: app/styles/@tradegecko/warehousing/components/_button.scss [hunk 4]
```diff
@@ -1,82 +1,90 @@
...
   text-align: center;
-  border: 1px solid;
   border-radius: $button-border-radius;
-  text-transform: uppercase;
   cursor: pointer;
 
   &[disabled] {
-    cursor: default;
-    pointer-events: none;
+    opacity: 0.4;
```
Comment (selvagsz) [inline app/styles/@tradegecko/warehousing/components/_button.scss:16]: should the cursor be changed to default or disabled ?

File: app/styles/@tradegecko/warehousing/components/_button.scss [hunk 5]
```diff
@@ -1,82 +1,90 @@
...
+  }
+
+  &--primary {
+    font-size: $button-font-size;
+    line-height: $button-line-height;
+    color: $white;
+    background-color: $ui-green-dark;
+    border: 0;
+    padding: 10px 20px;
```
Comment (selvagsz) [inline app/styles/@tradegecko/warehousing/components/_button.scss:25]: inconsistent padding btw primary & secondary.

Comment (abdullacontractor) [reply app/styles/@tradegecko/warehousing/components/_button.scss:25]: Yep, intentional, secondary has a border - primary does not

File: app/styles/@tradegecko/warehousing/components/_button.scss (hunk 4 above)
Comment (abdullacontractor) [reply app/styles/@tradegecko/warehousing/components/_button.scss:16]: cursor changes to default already

File: app/styles/@tradegecko/warehousing/variables/base-colors.scss (hunk 2 above)
Comment (abdullacontractor) [reply app/styles/@tradegecko/warehousing/variables/base-colors.scss:77]: Yep, checked

File: tests/dummy/app/index.html (hunk 1 above)
Comment (abdullacontractor) [reply tests/dummy/app/index.html:15]: Yep, change has been made on rango and pacman

File: app/styles/@tradegecko/warehousing/components/_button.scss (hunk 5 above)
Comment (selvagsz) [reply app/styles/@tradegecko/warehousing/components/_button.scss:25]: better to have a transparent border in that case. But not a blocker

**Review by selvagsz — APPROVED** [review]

File: tests/dummy/app/index.html (hunk 1 above)
Comment (billybonks) [reply tests/dummy/app/index.html:15]: by object id i think you mean `d3ds1rcygteua7` this is not an s3 thing but the cloudfront distribution id. it wont change.

**Review by billybonks — APPROVED** [review]
I will probably steal your button styles from here :).

**Pull request merged**